workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+
+
+```python
+speed(0)
+
+bg = turtle.Screen()
+bg.bgcolor("#99004D") #Setting back ground color
+
+y = -100
+width = 240
+
+```
+******speed()****** function determines the speed of the turtle, ie, the speed in which the figures get drawn on the screen. When we set that to zero it moves as fast as it can. 
+Next we set the background color to maroon using th hexcode.
+Now decide a value for ******y******, which is basically trial and error. Set some random value first and then iterate to find the right vaue which brings your turtle to the center.
+In Case of ******width******, provide any value you desire. 
+
+```python
+# Constructing the tree
+box1(turtle, "#4F4553", -15, y - 40, 30, 40) #Stem of the tree
+```
+Now we make a make a stem for the tree by passing desired values for each parameter. Color will be a shade of brown and the rest as you see above. 
+![alt text](https://cloud-4ipo9natc.vercel.app/4image3.png) 
+
+
+Next thing we are gonna make is the tree leaves with balls on either sides.
+This is the only part you have to be a bit careful while doing. See if the code is undertandable.
+
+```python
+while width > 20 :
+    width = width - 20
+    height = randint(20,30)
+    x = 0 - width/2
+    box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+    balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+    balls(turtle, "#fc045b", -x, y, 5) # Decorative balls
+    y = y + height
+
+```
+Not too difficult, is it ? Let's break this down.
+
+
+```python
+width = width - 20
+height = randint(20,30)
+```
+In each iteration, the width of the next rectangle gets reduced to a smaller size so that in the end we get a triangle kinda look for the entire leaf part (green portion). We assign a random value for the height as that helps to give the tree an irregular shape rather than a giving same height for all the rectangles. If you would like, you could reduce the width by a random integer too. 
+
+
+```python
+x = 0 - width/2
+```
+This is just to set the starting 'x' coordinate.
+
+```python
+box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+```
+Now we draw one complete green color filled box for first layer of flowers.
+![alt text](https://cloud-4ipo9natc.vercel.app/5image4.png) 
+
+```python
+balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+```
+Since the turtle is back on the left end of the rectangle, this statement draws a small ball of radius 5 with a shade of red filled in it. 
+![alt text](https://cloud-4ipo9natc.vercel.app/6image5.png) 
+Now we need one more tiny ball on the right end of the christmas tree right ? 
+
+```python
+balls(turtle, "#fc045b", -x, y, 5)
+```
+So we draw another ball with the same dimentions but we pass a different value for x coordinate. We pass ******-x****** instead of ******x******. This helps to position the ball on the right end instead of the left. 
+![alt text](https://cloud-4ipo9natc.vercel.app/7image6.png) 
```

Comment:
```suggestion
So we draw another ball with the same dimensions but we pass a different value for x coordinate. We pass ******-x****** instead of ******x******. This helps to position the ball on the right end instead of the left. 
![alt text](https://cloud-4ipo9natc.vercel.app/7image6.png) 
```